### ISSUE TITLE
At the Model Level, Standardize both *within* tasks and *across* tasks

### ISSUE THREAD

**Issue by xehu** [Watts-Lab/team-process-map#125]
Currently, in our model_builder, we standardize features after the entire multi-task dataframe has been assembled; that is, after we append data from multiple tasks into a single dataframe, we then normalize each column:

```
# Feature normalization - not relevant for tree based models like XGB or RF. But will cause significant differences in the case of linear models (Linear, Lasso, etc.)
        X = pd.DataFrame(StandardScaler().fit_transform(X),columns = X.columns)
```

However, in standardizing across all columns, we capture a lot less *within-task variation*, while capturing more *between-task variation* in these features.

Consider the case of speaking time. Juries, in general, speak a lot more than teams playing CSOP. So, by standardizing across all tasks, any jury will end up with a relatively high z-score; any CSOP team --- even the most talkative CSOP team of the lot --- will have a relatively low CSOP score.

One the one hand, standardizing across all columns has benefits: it helps identify these different trends across tasks (which is good for visualization, and sanity checking!).

As a result, we have a harder time identifying how being a *relatively* less talkative Jury, or a *relatively* more talkative CSOP team, might impact your performance. We lose some predictive power at the task level.

# The Proposal
Add an option for standardization to be either within- or across- tasks, so that we can test both in models. Will we get better performance if we standardize within tasks?
